### PR TITLE
fix/yolov5_example_type_inaccuracies

### PR DESCRIPTION
**Pull request by SkalskiP**
I noticed some type inconsistencies in your YOLOv5 example and decided to create a small PR with the fix. Mostly changing `Tuple[int]` -> `Tuple[int, int]`, but also `torch.Tensor` -> `Union[torch.Tensor, numpy.ndarray]` in one place.

### CODE REVIEW

File: examples/ultralytics-yolo/deepsparse_utils.py [hunk 1]
```diff
@@ -368,7 +368,7 @@ class YoloPostprocessor:
         output shapes
     """
 
-    def __init__(self, image_size: Tuple[int] = (640, 640), cfg: Optional[str] = None):
+    def __init__(self, image_size: Tuple[int, int] = (640, 640), cfg: Optional[str] = None):
```
Comment (bfineran) [inline examples/ultralytics-yolo/deepsparse_utils.py:371]:
```suggestion
    def __init__(
        self,
        image_size: Tuple[int, int] = (640, 640),
        cfg: Optional[str] = None,
    ):
```

File: examples/ultralytics-yolo/deepsparse_utils.py [hunk 2]
```diff
@@ -838,7 +838,7 @@ def _non_max_suppression(
     return output
 
 
-def _xywh2xyxy(x):
+def _xywh2xyxy(x: Union[torch.Tensor, numpy.ndarray]) -> Union[torch.Tensor, numpy.ndarray]:
```
Comment (bfineran) [inline examples/ultralytics-yolo/deepsparse_utils.py:841]:
```suggestion
def _xywh2xyxy(
    x: Union[torch.Tensor, numpy.ndarray],
) -> Union[torch.Tensor, numpy.ndarray]:
```

**Review by bfineran — COMMENTED** [review]
thanks for the PR @SkalskiP! Looks good overall, left two suggestions to stay within our style guidelines for line length. (run `make quality` to check)

**Comment by SkalskiP** [comment]
Sure @bfineran I'll make sure the line lengths are correct.

**Comment by SkalskiP** [comment]
@bfineran changes are done, please take a look

**Review by mgoin — APPROVED** [review]
nice catches, lgtm thanks!

**Review by markurtz — APPROVED** [review]

**Pull request merged**